control-plane/rest/service/src/v0/nexuses.rs

```diff
@@ -114,13 +114,19 @@ impl apis::actix_server::Nexuses for RestApi {
 
     async fn put_node_nexus_share(
         Path((node_id, nexus_id, protocol)): Path<(String, Uuid, models::NexusShareProtocol)>,
+        Body(share_nexus_body): Body<models::ShareNexusBody>,
     ) -> Result<String, RestError<RestJsonError>> {
-        let share = ShareNexus {
-            node: node_id.into(),
-            uuid: nexus_id.into(),
-            key: None,
-            protocol: protocol.into(),
-        };
+        // let share = ShareNexus {
```

Comment:
Is this commented code supposed to be removed?

Reply:
I thought i removed .Ill check

Reply:
Incorporated